pages/app-layout/utils/iframe-wrapper.tsx

```diff
@@ -31,15 +29,26 @@ function syncClasses(from: HTMLElement, to: HTMLElement) {
 }
 
 export function IframeWrapper({ id, AppComponent }: { id: string; AppComponent: React.ComponentType }) {
-  const { urlParams } = useContext(AppContext);
-  const iframeRef = useRef<HTMLIFrameElement>(null);
+  const ref = useRef<HTMLDivElement>(null);
 
   useEffect(() => {
-    const iframeEl = iframeRef.current;
-    if (!iframeEl || !iframeEl.contentDocument) {
+    const container = ref.current;
+    if (!container) {
       return;
     }
-    const iframeDocument = iframeEl.contentDocument;
+    const iframeEl = container.ownerDocument.createElement('iframe');
+    iframeEl.className = styles['full-screen'];
+    iframeEl.id = id;
+    iframeEl.title = id;
+    iframeEl.srcdoc = '<!DOCTYPE html>';
+    container.appendChild(iframeEl);
+
+    const iframeDocument = iframeEl.contentDocument!;
+    // Prevent iframe document instance from reload
+    // https://bugzilla.mozilla.org/show_bug.cgi?id=543435
+    iframeDocument.open();
+    iframeDocument.close();
```

Comment:
Main lines of the fix. Among some alternatives I picked this one, because this is what our production customers do as well